src/modules/launcher/Wox.Infrastructure/Storage/JsonStorage`1.cs

```diff
@@ -106,7 +106,7 @@ private void LoadDefault()
 
         private void BackupOriginFile()
         {
-            var timestamp = DateTime.Now.ToString("yyyy-MM-dd-HH-mm-ss-fffffff", CultureInfo.CurrentUICulture);
```

Comment:
Any reason for changing this? I don't think we should change it to maintain backward compatibility.

Reply:
Fxcop complains about CurrentUICulture

`CA1305	'JsonStorage<T>.BackupOriginFile()' passes 'CultureInfo.CurrentUICulture' as the 'IFormatProvider' parameter to 'DateTime.ToString(string, IFormatProvider)'. This property returns a culture that is inappropriate for formatting methods.`

Reply:
Sounds good. Taking a deeper look at the code it seems like that value `timestamp` is being used to append to a corrupt json file. It shouldn't matter which culture we use.